wordpress.org/public_html/wp-content/mu-plugins/pub/wporg-robots.php

```diff
@@ -20,9 +20,12 @@ function wporg_robots_txt( $robots ) {
 		           "Allow: /locale/*/stats/plugins/$\n" .
 		           "Allow: /locale/*/stats/themes/$\n";
 
-	} elseif ( 'wordpress.org' === $blog_details->domain ) {
+	} elseif ( 'wordpress.org' === $blog_details->domain || 'developer.wordpress.org' === $blog_details->domain ) {
 		// WordPress.org/search/ should not be indexed.
 		$robots .= "\nUser-agent: *\n" .
+				   "Disallow: /wp-admin/\n" .
```

Comment:
Adding that will revert https://github.com/WordPress/wordpress.org/commit/7cd49035860c0614b05861ffef1d95796c593e1d /  https://meta.trac.wordpress.org/ticket/5806

If we no longer want load-scripts/styles to be accessible, we should remove it globally than just on the homepage and developer.

Reply:
Actually, the new theme doesn't seem to require load-scripts.php, as jQuery isn't loaded anymore.. so I guess the later str_replace could change instead.